### PR TITLE
Do not use sudo when setting up the service

### PR DESCRIPTION
**Pull request by mtpereira**
Instead of relying on `sudo` (which at the moment is only used for operating docker), the user running the application can be added to the `docker` group.

### CODE REVIEW

**Comment by mtpereira** [comment]
:warning: Let's only merge this once we've got confirmation it is working on the new setup (I see no reason for not to, but we all know how it goes :upside_down_face:).

**Review by brunoamaral — APPROVED** [review]

**Comment by brunoamaral** [comment]
Thank you! There are also some mentions of node-red and metabase that are causing errors. I will remove them after this merge.

**Pull request merged**
